services/core-web/src/components/modalContent/AddAdministrativeAmendmentModal.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Highlight from "react-highlighter";
+import AdministrativeAmendmentForm from "@/components/Forms/noticeOfWork/AdministrativeAmendmentForm";
+
+const propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+};
+
+const defaultProps = {};
+
+export const AddAdministrativeAmendmentModal = (props) => (
+  <div>
+    <p>
+      Creating an <Highlight search="Administrative Amendment">Administrative Amendment</Highlight>{" "}
+      allows you to update/change the selected permit amendment.
```

Comment:
Did we want to reduce this to just "update", instead of "update/change"?

Reply:
It would be nice to get it all on one line:
![image](https://user-images.githubusercontent.com/17113094/112698587-33559100-8e47-11eb-8771-7cf413f06e28.png)

Reply:
updated it to be "update"